pyspectral/utils.py

```diff
@@ -244,7 +244,7 @@ def sort_data(x_vals, y_vals):
     return x_vals, y_vals
 
 
-def convert2hdf5(ClassIn, platform_name, bandnames, scale=1e-06):
+def convert2hdf5(ClassIn, platform_name, bandnames, scale=1e-06, dets=None):
```

Comment:
I think the new kwarg should be `detectors` for clarity. And have a short explanation in the docstring.